src/shared/proto/robocup_extended.proto

```diff
@@ -0,0 +1,116 @@
+syntax = "proto3";
```

Comment:
This file shopulent be necessary. It was only done to show that teams can arbitrarily add extensions to the protocol without impacting the main protocol.

Reply:
Ah, gotcha